host/elasticsearch_test.go

```diff
@@ -1137,14 +1137,24 @@ func (s *elasticsearchIntegrationSuite) TestUpsertWorkflowExecution() {
 		history *historypb.History,
 	) ([]*commandpb.Command, error) {
 
-		upsertCommand := &commandpb.Command{
-			CommandType: enumspb.COMMAND_TYPE_UPSERT_WORKFLOW_SEARCH_ATTRIBUTES,
-			Attributes: &commandpb.Command_UpsertWorkflowSearchAttributesCommandAttributes{
-				UpsertWorkflowSearchAttributesCommandAttributes: &commandpb.UpsertWorkflowSearchAttributesCommandAttributes{},
-			},
+		var upsertCommand *commandpb.Command
+		if commandCount < 3 {
+			upsertCommand = &commandpb.Command{
+				CommandType: enumspb.COMMAND_TYPE_UPSERT_WORKFLOW_SEARCH_ATTRIBUTES,
+				Attributes: &commandpb.Command_UpsertWorkflowSearchAttributesCommandAttributes{
+					UpsertWorkflowSearchAttributesCommandAttributes: &commandpb.UpsertWorkflowSearchAttributesCommandAttributes{},
+				},
+			}
+		} else {
+			upsertCommand = &commandpb.Command{
```

Comment:
Please create separate test for `UpsertMemo`. Don't mix them with search attributes.